src/ScottPlot/Plottable/ScatterPlot.cs

```diff
@@ -295,7 +295,8 @@ public LegendItem[] GetLegendItems()
                 lineStyle = LineStyle,
                 lineWidth = LineWidth,
                 markerShape = MarkerShape,
-                markerSize = MarkerSize
+                markerSize = MarkerSize,
+                plottableVisible = IsVisible
```

Comment:
This makes sense... but if we go with this design I'm guessing we would have to do this for each `GetLegendItems()` method of _every_ plottable 😅

Looks like there are about 30 places: https://github.com/ScottPlot/ScottPlot/search?q=GetLegendItems

Reply:
That's why I requested you input, I did not see another way to handle this, but maybe you will. For the moment, that is why I restricted the modification to `Scatter` alone

Reply:
https://github.com/ScottPlot/ScottPlot/pull/1589#discussion_r792273087